internal/store/continuation_repo.go

```
@@ -0,0 +1,216 @@
+	rows, err := r.db.sql.Query(
+		`SELECT id, auto_retry_reason
+		 FROM tasks
+		 WHERE status = ? AND auto_retry_state = ?
+		 ORDER BY id`,
+		domain.TaskFailed,
+		domain.ContinuationStopped,
+	)
+	if err != nil {
+		return nil, fmt.Errorf("query stopped continuations: %w", err)
+	}
+	defer rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)